Evergreen/MainWindow/MainWindowController.swift

```diff
@@ -89,6 +97,13 @@ class MainWindowController : NSWindowController, NSUserInterfaceValidations {
 			openInBrowser(link)
 		}		
 	}
+    
+    @IBAction func showShareWindow(_ sender: AnyObject) {
+        if let link = currentLink, let url = URL(string: link) {
+            let picker = NSSharingServicePicker(items: [url])
+            picker.show(relativeTo: sender.bounds, of: sender as! NSView, preferredEdge: NSRectEdge.minY)
+        }
```

Comment:
`sender` should shouldn't be force unwrapped.  All of the parameters to the picker can be safely unwrapped before being used.

```swift
if let link = currentLink,
    let url = URL(string: link)
    let sender = sender as? NSView {
        
    let picker = NSSharingServicePicker(items: [url])
    picker.show(relativeTo: sender.bounds, of: sender , preferredEdge: NSRectEdge.minY)
}
```